packages/core/src/Param/Param.ts

```diff
@@ -9,6 +8,12 @@ import { jsExpressionEvaluation } from './evaluations/jsExpressionEvaluation'
 import { jsFunctionEvaluation } from './evaluations/jsFunctionEvaluation'
 import { jsonEvaluation } from './evaluations/jsonEvaluation'
 
+export interface StringableInputValue extends Record<string, any> {
+  value: any,
+  Evaluation?: string,
```

Comment:
Is there a reason Evaluation and Cast have initial capitalization?

Reply:
This is dynamically created from `optionGroups`. I've selected the `label` of the `optionGroup` as the key for saving the form. 

![image](https://github.com/ajthinking/data-story/assets/20497176/44333ffa-71cc-49e2-8d2b-e761ccf60d5a)